handbook/engineering/hiring/engineering-manager-backend-platform.md

```diff
@@ -0,0 +1,72 @@
+# Engineering Manager - Backend Platform
+
+We are looking for an engineering manager to join the team that is building the backend platform upon which product engineering teams can iterate quickly and safely. The team's mission is to promote and champion consistency and simplicity across all backend code by building shared tools, libraries and infrastructure for common use cases, scale and maintain said infrastructure and ultimately create leverage for and accelerate other teams.
+
+The [Cloud team](https://about.sourcegraph.com/handbook/engineering/cloud) team will become the Backend Platform team once we hire a Product Manager to drive the Cloud project forward.
+
+## About you
+
+- You know how to authentically inspire, challenge, and grow individuals and teams.
+- You are motivated to work on ambitious long term goals to solve complex technical challenges.
+- You can effectively reason about and communicate the value of platform projects to stakeholders.
+- You thrive in making plans with your team, and communicate those appropriately.
+- You are [high agency](https://www.linkedin.com/pulse/high-agency-its-importance-how-cultivate-shreyas-doshi/).
+
+## Your responsibilities
+
+- You will facilitate a cohesive team environment where everyone is happy, motivated, and productive.
+- You will ensure that the team has [well defined goals that are aligned with the business](https://about.sourcegraph.com/handbook/engineering/cloud#goals) that are effectively communicated to all levels of the company, and that the team is making continuous iterative progress towards those goals every week.
+- You will help engineers grow in their careers.
+- You will hire exceptional engineers to grow the team in a sustainable way so that the team can accomplish more over time.
```

Comment:
I want to work on merging and aligning this with https://github.com/sourcegraph/about/pull/2075. Will propose some edits.